usr/share/rear/layout/save/GNU/Linux/210_raid_layout.sh

```diff
@@ -15,14 +16,20 @@ if [ -e /proc/mdstat ] &&  grep -q blocks /proc/mdstat ; then
             fi
 
             # We use the detailed mdadm output quite a lot
-            mdadm --misc --detail $device > $TMP_DIR/mdraid
+            tmpfile="$TMP_DIR/mdraid.$name"
```

Comment:
The variable name "tmpfile" is not really meaningful
because it only tells an implementation detail (it is a temporary file)
but it does not tell what that thingy actually is about
so a better name could be e.g. "mdadm-details".